### PR TITLE
Update composer deps

### PR DESCRIPTION
**Pull request by nikola-jovanovic-php**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,12 +15,12 @@
...
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": "^8.1",
         "ext-mbstring": "*",
         "psr/http-client": "^1.0",
         "psr/http-factory": "^1.0",
         "psr/http-message": "^1.0",
-        "symfony/options-resolver": "^3.4 || ^4.0 || ^5.0"
+        "symfony/options-resolver": "^6.0"
```
Comment (soullivaneuh) [inline composer.json:23]: Please keep support of Symfony 4.4 and 5.4 LTS releases. We are following the official roadmap: https://symfony.com/releases#symfony-releases-calendar

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,5 @@
 /.phpunit.result.cache
 /composer.lock
 /vendor/
+/.idea/
```
Comment (soullivaneuh) [inline .gitignore:5]: IDE specific ignore rule that should not be present on this project.

File: .travis.yml [hunk 3]
```diff
@@ -1,50 +0,0 @@
-branches:
```
Comment (soullivaneuh) [inline .travis.yml:1]: The Travis configuration should not be removed. Why this action?

File: psalm.xml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<psalm
```
Comment (soullivaneuh) [inline psalm.xml:2]: Is it an equivalent of PHPStan? If yes, we would prefer using that tool.

File: src/Attachment.php [hunk 5]
```diff
@@ -493,24 +457,24 @@ public function clearActions(): self
     public function toArray(): array
     {
         $data = [
-            'fallback' => $this->fallback,
-            'text' => $this->text,
-            'pretext' => $this->pretext,
-            'color' => $this->color,
-            'footer' => $this->footer,
+            'fallback'    => $this->fallback,
```
Comment (soullivaneuh) [inline src/Attachment.php:460]: Array alignment style is not desired, please revert.

Note: This request change has to be applied globally.

File: rector.php [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+use Rector\Config\RectorConfig;
+use Rector\Set\ValueObject\LevelSetList;
+use Rector\Set\ValueObject\SetList;
+use Rector\Symfony\Set\JMSSetList;
+use Rector\Symfony\Set\SensiolabsSetList;
+use Rector\Symfony\Set\SymfonyLevelSetList;
+use Rector\Symfony\Set\SymfonySetList;
+use Rector\TypeDeclaration\Rector\FunctionLike\ReturnTypeDeclarationRector;
+
+return static function (RectorConfig $rectorConfig): void {
```
Comment (soullivaneuh) [inline rector.php:14]: Can this tool be run under a CI job?

**Review by soullivaneuh — CHANGES_REQUESTED** [review]
Thanks for you contribution! I put some changes request on the diff.

**Comment by nikola-jovanovic-php** [comment]
@soullivaneuh sorry for that, i'm going to close this PR. This is meant to be for my organization fork, as we were moving to symfony 6.1 and official package weren't supporting that version yet. Didn't notice that i opened PR to wrong repo :(

**Pull request closed without merging**